app/models/inning.rb

```diff
@@ -41,13 +74,40 @@ def add_two_outs
     end
   end
 
+  def two_outs
+    outs == 2
+  end
+
+  def retire_the_side
+    clear_bases
+    self.update(outs: 3)
+  end
+
+  def runners_advance_one_base
```

Comment:
What is this? Future method? Should we build it out and keep it? Or remove it for now?